dotcom-rendering/src/lib/getTagFrontsAdPositions.ts

```diff
@@ -0,0 +1,68 @@
+import type { GroupedTrailsBase } from '../types/tagFront';
+import {
+	getMerchHighPosition,
+	MAX_FRONTS_BANNER_ADS,
+	MAX_FRONTS_MOBILE_ADS,
+} from './getFrontsAdPositions';
+
+const hasAdjacentCommercialContainer = (
+	collectionIndex: number,
+	merchHighPosition: number,
+) => collectionIndex === merchHighPosition;
+
+const isEvenIndex = (_collection: unknown, index: number) => index % 2 === 0;
+
+/**
+ * Uses a very similar approach to pressed fronts, except we
+ * - Do not need to consider thrashers
+ * - Do not need to consider the 'most viewed' container
+ *
+ * The types are also slightly different, as we no longer have
+ * specific container IDs, so we use the date which is unique
+ */
+const getTagFrontMobileAdPositions = (
+	collections: Array<GroupedTrailsBase>,
+): number[] => {
+	const merchHighPosition = getMerchHighPosition(collections.length);
+
+	return collections
+		.filter(
+			(_, index) =>
+				!hasAdjacentCommercialContainer(index, merchHighPosition),
+		)
+		.filter(isEvenIndex)
+		.map((collection) =>
+			collections.findIndex(
+				({ day, month, year }) =>
+					day === collection.day &&
+					month === collection.month &&
+					year === collection.year,
+			),
+		)
+		.filter((adPosition: number) => adPosition !== 1)
```

Comment:
What about `-1`s?

```suggestion
		.filter((adPosition: number) => adPosition > 1)
```

Reply:
Yeah this is just a mistake. Thanks for catching! 🙏